ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  ansible-operator:
+    name: "4.9"
+    namespace: ocp
+    tag: ansible-operator
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
```

Comment:
Wouldn't we want this to be 8 vs. 7?

Reply:
Tested with centos:8 but looks like it is not available.

Reply:
There's `centos:8` in `origin` namespace.